test/cmdlineTests/structured_documentation_source_location/err

```diff
@@ -0,0 +1,11 @@
+Error: Documentation tag "@return No value returned" does not contain the name of its return parameter.
+ --> structured_documentation_source_location/input.sol:3:5:
+  |
+3 |     /// @param id Some identifier
+  |     ^ (Relevant source part starts here and spans across multiple lines).
+
+Error: Documentation tag "@return No value returned" does not contain the name of its return parameter.
+ --> structured_documentation_source_location/input.sol:7:5:
+  |
+7 |     /// @return No value returned
```

Comment:
I don't understand yet, why the syntax test for a single line structured documentation block passes, but here the formatter thinks that it spans over multiple lines.